v0/destinations/marketo/util.js

```diff
@@ -91,14 +104,13 @@ const marketoResponseHandler = (destResponse, sourceMessage, stage) => {
   }
   // non 2xx failure
   throw new ErrorBuilder()
-    .setStatus(status)
+    .setStatus(400)
```

Comment:
Let's set the stage as well if it's not defaulted already.

Reply:
The stage property is being sent as an argument and that is being set as part of `setStatTags`